x-pack/plugins/actions/tsconfig.json

```diff
@@ -47,7 +47,8 @@
     "@kbn/core-http-server",
     "@kbn/core-test-helpers-kbn-server",
     "@kbn/security-plugin-types-server",
-    "@kbn/core-application-common"
+    "@kbn/core-application-common",
+    "@kbn/alerting-plugin"
```

Comment:
Might be easiest to copy the `retryIfConflicts` fn into the actions plugin for now to avoid circular dependencies. I don't know if there's a package that it would make sense to go into